sdk-extension/sdk-src/src/main/java/com/hpe/adm/nga/sdk/extension/network/ResponseInterceptor.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 Hewlett-Packard Enterprise Development Company, L.P.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hpe.adm.nga.sdk.extension.network;
+
+import java.util.Map;
+
+/**
+ * Default implementation, useful if you want to override just one of the methods
+ */
+public interface ResponseInterceptor {
+    /*
+    TODO: disabled content interceptor, google http response not mutable, to discuss
```

Comment:
discussing?

Reply:
I just don't have time to deal with this right now, sadly atm you can only change the headers of the response

Reply:
I'll find the time to fix this, for next release